tests/vmi_multiqueue_test.go

```diff
@@ -123,5 +123,35 @@ var _ = Describe("[Serial]MultiQueue", func() {
 			}
 		})
 
+		It("should be able to create a multi-queue VMI when requesting a single vCPU", func() {
+			vmi := tests.NewRandomFedoraVMIWitGuestAgent()
+			multiQueue := true
+
+			// by default, we request a single vCPU
+			vmi.Spec.Domain.Devices.NetworkInterfaceMultiQueue = &multiQueue
+			vmi.Spec.Domain.Devices.Rng = &v1.Rng{}
+
+			By("Creating and starting the VMI")
+			vmi, err := virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
+			Expect(err).ToNot(HaveOccurred())
+			tests.WaitForSuccessfulVMIStartWithTimeout(vmi, 360)
+
+			By("Checking if we can login")
+			expecter, err := tests.LoggedInFedoraExpecter(vmi)
+			defer expecter.Close()
```

Comment:
As a general rule, first errors need to be checked before close-defers are created. If errors happen return values are very often nil.

Reply:
As @EdDev pointed out, this is not required for anything within the test.

I removed it in the last push.